tests/travis-ci/log-express-performance.php

```diff
@@ -69,5 +90,51 @@
   // print_r('Query Count: ' . $query_count_average . " Queries\n");
   // print_r('Query Time: ' . $query_time_average . " Milliseconds\n");
   print_r("\n");
-  $i++;
+
+  // Prep data per test to send to logstash
+  $data = array(
+    'test_type' => 'indvidual_test',
+    'path' => $path['path'],
+    'accessed' => $count,
+    'memory_consumption' => $memory_average,
+    'loadtime' => $load_average,
+    'build_branch' => $build_branch,
+    'build_number' => $build_number,
+    'build_type' => $build_type,
+  );
+ curl_logstash($data);
+}
+
+function curl_logstash($data) {
+  $data_string = json_encode($data);
+
+  # If data is not getting into the logging stack, check the IP range of the
+  # TravisCI env. https://docs.travis-ci.com/user/ip-addresses/.
+  # Currently is
+  # workers-nat-org-shared-2.aws-us-east-1.travisci.net (52.45.185.117/32 52.54.31.11/32 54.87.185.35/32 54.87.141.246/32)
+  $ch = curl_init();
+  curl_setopt($ch, CURLOPT_URL, "http://wlogstash.colorado.edu:8080");
+  curl_setopt($ch, CURLOPT_CUSTOMREQUEST, "POST");
+  curl_setopt($ch, CURLOPT_POSTFIELDS, $data_string);
+  curl_setopt($ch, CURLOPT_TIMEOUT, 10);
+  curl_setopt($ch, CURLOPT_HTTPHEADER, array('Content-Type: application/json','User-Agent: TravisCI'));
+
+  if( ! $result = curl_exec($ch)) {
+    print_r("cURL error: " . curl_error($ch) . "\n\n");
+    $starttime = microtime(true);
+    $file      = fsockopen ('wlogstash.colorado.edu', 8080, $errno, $errstr, 10);
```

Comment:
I assume $errno and $errstr are created from cURL receiving an error, but I don't see those variables declared in this file.